_api-reference/cat/cat-segment-replication.md

```diff
@@ -39,10 +39,11 @@ Parameter | Data type | Description
 `active_only` | Boolean | If `true`, the response only includes active segment replications. Defaults to `false`. 
 [`detailed`](#additional-detailed-response-metrics) | String | If `true`, the response includes additional metrics for each stage of a segment replication event. Defaults to `false`.
 `shards` | String | A comma-separated list of shards to display.
+`byte` | Byte size | [Units]({{site.url}}{{site.baseurl}}/opensearch/units/) used to display byte size values.
```

Comment:
Is there a default for this parameter? Since it's optional, I assume there should be a default units to display byte values.

Reply:
```suggestion
`byte` | Byte units | [Units]({{site.url}}{{site.baseurl}}/opensearch/units/) used to display byte size values.
```

Reply:
Sorry to show the Elastic's document https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cat.html#numeric-formats, but that can make the explanation easier.

By default, the value of "bytes" and "time" fields are formatted to be human-readable. The value will be added with the most suitable unit.
For example:
When file size larger than 1000 byte, the unit will be shown as "kb", otherwise shown as "b". 
When the time is more than 1000 milliseconds, the unit will be shown as "s", otherwise shown as "ms".
When the number is even larger, a higher-level unit will be shown.
```
$ curl "opens-clust-xxx.elb.us-west-2.amazonaws.com/_cat/segment_replication?v"
shardId          target_node                              target_host checkpoints_behind bytes_behind current_lag last_completed_lag rejected_requests
[logs-221998][7] ip-10-0-4-86.us-west-2.compute.internal  10.0.4.86   1                  3b       988ms       823ms              0
[logs-221998][7] ip-10-0-3-170.us-west-2.compute.internal 10.0.3.170  1                  39.2kb       988ms       1.2s               0
[logs-231998][0] ip-10-0-3-66.us-west-2.compute.internal  10.0.3.66   1                  31.6kb       791ms       699ms              0
```
This is an example of the API response when assigning the parameters `time=ms` and `bytes=b`, the value will be purely numeric and directly comparable.
```
$ curl "opens-clust-xxx.elb.us-west-2.amazonaws.com/_cat/segment_replication?v&time=ms&bytes=b"
shardId          target_node                              target_host checkpoints_behind bytes_behind current_lag last_completed_lag rejected_requests
[logs-221998][0] ip-10-0-5-102.us-west-2.compute.internal 10.0.5.102  1                  7480         1116        1664               0
[logs-221998][0] ip-10-0-3-36.us-west-2.compute.internal  10.0.3.36   1                  16034        1116        1316               0
[logs-221998][0] ip-10-0-4-86.us-west-2.compute.internal  10.0.4.86   1                  16034        1116        2018               0
```